source/website/src/components/Translation.vue

```diff
@@ -537,7 +531,7 @@ export default {
           .sort(function(a, b) {
             const textA = a.text.toUpperCase();
             const textB = b.text.toUpperCase();
-            return (textA < textB) ? -1 : (textA > textB) ? 1 : 0;
+            return (textA < textB && -1) || (textA > textB && 1) || 0;
```

Comment:
does this increase cognitive complexity?

Reply:
Not according to the algorithm it uses for cognitive complexity.

Personally, I think both approaches are just as clear to understand but I'd guess for most people who are new to JavaScript the original code is probably more clear. However, SonarQube didn't like it. It rejects the nested `? :` operators. But, for some reason, it is perfectly happy with the alternate approach.
It's an arbitrary rule, in my opinion, but it is what it is.